lib/spack/spack/test/web.py

```diff
@@ -100,47 +100,47 @@ def test_spider_no_response(monkeypatch):
 @pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_find_versions_of_archive_0():
     versions = spack.util.web.find_versions_of_archive(root_tarball, root, list_depth=0)
-    assert ver("0.0.0") in versions
+    assert ver("=0.0.0") in versions
 
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_find_versions_of_archive_1():
     versions = spack.util.web.find_versions_of_archive(root_tarball, root, list_depth=1)
-    assert ver("0.0.0") in versions
-    assert ver("1.0.0") in versions
+    assert ver("=0.0.0") in versions
+    assert ver("=1.0.0") in versions
 
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_find_versions_of_archive_2():
     versions = spack.util.web.find_versions_of_archive(root_tarball, root, list_depth=2)
-    assert ver("0.0.0") in versions
-    assert ver("1.0.0") in versions
-    assert ver("2.0.0") in versions
+    assert ver("=0.0.0") in versions
+    assert ver("=1.0.0") in versions
+    assert ver("=2.0.0") in versions
 
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_find_exotic_versions_of_archive_2():
     versions = spack.util.web.find_versions_of_archive(root_tarball, root, list_depth=2)
     # up for grabs to make this better.
-    assert ver("2.0.0b2") in versions
+    assert ver("=2.0.0b2") in versions
 
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_find_versions_of_archive_3():
     versions = spack.util.web.find_versions_of_archive(root_tarball, root, list_depth=3)
-    assert ver("0.0.0") in versions
-    assert ver("1.0.0") in versions
-    assert ver("2.0.0") in versions
-    assert ver("3.0") in versions
-    assert ver("4.5") in versions
+    assert ver("=0.0.0") in versions
+    assert ver("=1.0.0") in versions
+    assert ver("=2.0.0") in versions
+    assert ver("=3.0") in versions
+    assert ver("=4.5") in versions
```

Comment:
its making me uneasy that every use of `in` is going to need an `=`.  I think we should talk about this one and figure out if this one can be fixed... I'm not seeing an easy way though... maybe it's fine.

Can't these be changed to `Version()` to make them more clear?  Maybe the issue I'm having here is that the tests use `ver` all over the place and we should just be using `Version` for concrete stuff.

Reply:
Directly using `Version(...)` sounds good to me, it's clear